cytominer_database/commands/command_ingest.py

```diff
@@ -0,0 +1,55 @@
+import click
+import configparser
+import os
+import pkg_resources
+
+import cytominer_database.ingest
+import cytominer_database.munge
+
+
+@click.command(
+    "ingest",
+    help="""\
+Import CSV files into a database.
+
+SOURCE is a directory containing subdirectories that contain CSV files.
+"""
+)
+@click.argument(
+    "source",
+    type=click.Path(exists=True)
+)
+@click.option(
+    "-c",
+    "--config-file",
+    default=pkg_resources.resource_filename(
+        __name__,
+        os.path.join("config", "config_htqc.ini")
+    ),
+    help="Configuration file.",
+    type=click.Path(exists=True)
+)
+@click.option(
+    "--munge/--no-munge",
+    default=True,
+    help="""\
+True if the CSV files for individual compartments
+have been merged into a single CSV file. If True,
+the CSV will be split into one CSV per compartment.
+"""
+)
+@click.option(  # TODO: Is this an argument/required?
```

Comment:
@shntnu Is `--target` a required argument? It looks like there isn't a provided default. Should the command be:

```
cytominer-database [OPTIONS] SOURCE TARGET
```

Reply:
`--target` is required. The usage was specified incorrectly. 

```
Usage: cytominer-database ingest [OPTIONS] SOURCE -o TARGET
```